res/TensorFlowLiteRecipes/BroadcastTo_001/test.recipe

```diff
@@ -0,0 +1,28 @@
+# BroadcastTo of Float type is version 2 and Quantized BroadcastTo is version 3.
+# We are using tflite kernels for float values. so, the version should be 2.
+# refer https://github.com/tensorflow/tensorflow/pull/46224/files
```

Comment:
@seanshpark 

I misunderstood to remove code lines 1 to 4. 😅 
I modified it to 1~3 lines again, is it okay?

Reply:
> I modified it to 1~3 lines again, is it okay?

I don't quite understand your question.

Reply:
@seanshpark 
You gave comments to one line, but I misunderstood to delete the 1 to 3 code lines because of UI. 😅
I modified it again to reflect the comments you gave me, is it okay?

Reply:
Yes, looks OK. We have some trouble in communication. I hope it gets better next time.